src/main/java/com/github/tomakehurst/wiremock/core/Version.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2023 Thomas Akehurst
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.tomakehurst.wiremock.core;
+
+import java.io.IOException;
+import java.util.Properties;
+
+public class Version {
+  public static String getCurrentVersion() {
```

Comment:
How about caching it in a static variable? Java resource loading is heavy, and such unprotected endpoint may be exploited for various kinds of DDoS attempts

Reply:
Sounds great.  Thank you for the suggestion.  I have added simple caching of the version as suggested. If we add any other version information to the Version class we can add more sophisticated caching but for now this should do

Reply:
I've just merged a change with a `Lazy` utility class. This might be a nice way to do this (and is thread safe FWIW).

Reply:
Nice, I'll take a look at that and update the PR

Reply:
I have updated to the `Lazy` class.  Nice little utility class 👍